cpp/benchmarks/reduction/scan_structs.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <benchmarks/common/generate_input.hpp>
+#include <benchmarks/fixture/rmm_pool_raii.hpp>
+
+#include <nvbench/nvbench.cuh>
+
+#include <cudf/column/column_factories.hpp>
+#include <cudf/detail/scan.hpp>
```

Comment:
```suggestion
#include <cudf/column/column_factories.hpp>
#include <cudf/detail/scan.hpp>

#include <nvbench/nvbench.cuh>
```
nit: nvbench header is further than cudf ones